client/js/components/procedure/admin/AdministrationPlaces.vue

```diff
@@ -309,6 +315,19 @@ export default {
         })
     },
 
+    setEditMode (rowData, state = true) {
+      const idx = this.places.findIndex(el => el.id === rowData.id)
+
+      this.places[idx].edit = state
+    },
+
+    updatePlaceData (rowData) {
```

Comment:
same here. id instead of rowData

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/1487/commits/dce2bdabf97f182b0a38094e0af4228f22d8aec1

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/1487/commits/dce2bdabf97f182b0a38094e0af4228f22d8aec1